packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTable/TransactionsTableCustomAddressLabel.tsx

```diff
@@ -0,0 +1,19 @@
+import { MergedTransaction } from '../../../state/app/state'
+import { isCustomDestinationAddressTx } from '../../../state/app/utils'
+import { CustomAddressTxExplorer } from './TransactionsTable'
+
+export const TransactionsTableCustomAddressLabel = ({
+  tx
+}: {
+  tx: MergedTransaction
+}) => {
+  if (!isCustomDestinationAddressTx(tx)) {
+    return null
+  }
```

Comment:
I think I'd rather we do this check on the outside, meaning:

```
isCustomDestinationAddressTx(tx) && <TransactionsTableCustomAddressLabel tx={tx} />
```
That way it's more explicit from the outside to know when the component is rendering